ush/generate_HWT_ensemble.sh

```diff
@@ -12,7 +12,8 @@ for suite in FV3_GFS_v15_thompson_mynn rrfs_gfsv16 nssl_mp_no_nsst ; do
     sed s/__MEMBER__/$member/g $config_file > config.sh
     sed -i s/__CCPPSUITE__/$suite/g config.sh
 
-    ./generate_FV3LAM_wflow.sh
+    ./generate_FV3LAM_wflow.sh || exit 1
```

Comment:
I don't understand this

Reply:
Nevermind.  This is just to make the generation script exit right away if an error occurs when a workflow is generated instead of continuing on to generate the next workflow.  Right?  Sorry, long day.

Reply:
Yep, that's right. It was going through all 9 members even when the script failed and I didn't notice since there is SO MUCH screen output.